speech/speech_quickstart/main.go

```diff
@@ -29,7 +29,7 @@ func main() {
 	}
 
 	// Sets the name of the audio file to transcribe.
-	filename := "/path/to/audio.raw"
```

Comment:
I think these are fine, actually. For the quickstarts, we do expect people to modify the programs. I'd rather not make them too specific to this repo.

Reply:
ok! I'll revert to the original